config/src/config/safety_rules_config.rs

```diff
@@ -160,6 +167,37 @@ impl InitialSafetyRulesConfig {
             },
         }
     }
+
+    pub fn overriding_identity_blobs(&self) -> anyhow::Result<Vec<IdentityBlob>> {
+        match self {
+            InitialSafetyRulesConfig::FromFile {
+                overriding_identity_paths,
+                ..
+            } => {
+                let mut blobs = vec![];
+                for path in overriding_identity_paths {
+                    let blob = IdentityBlob::from_file(path)?;
+                    blobs.push(blob);
+                }
+                Ok(blobs)
+            },
+            InitialSafetyRulesConfig::None => {
+                bail!("loading overriding identity blobs failed with missing initial safety rules config")
+            },
+        }
+    }
+
+    pub fn overriding_identity_blob_paths_mut(&mut self) -> &mut Vec<PathBuf> {
```

Comment:
nit: Maybe make this test-only? It seems like it's only used in the smoke tests? Or better yet, don't expose this here if possible?

Reply:
fixed.

NOTE that we can't reuse feature `testing` as crate `smoke-test` needs it disabled in addition to `overriding_identity_blob_paths_mut`.

Reply:
Aah, in that case, don't worry about it -- you can remove the `smoke-test` feature. I was just wondering if it was a quick change 😄 Seems like overkill for a new feature.